tests/casts.cpp

```diff
@@ -392,6 +392,17 @@ TEST(fullConversion)/*{{{*/
 }/*}}}*/
 #endif // Vc_DEFAULT_TYPES
 
+TEST(referenceConstruction)
+{
+    float_v a = float_v::Random();
+
+    Vc::float_v r = a[2];
```

Comment:
Your ctor is `explicit`, so you need:
```suggestion
    Vc::float_v r = Vc::float_v(a[2]);
```
or:
```suggestion
    Vc::float_v r(a[2]);
```